src/util/check-github-app-type.ts

```diff
@@ -0,0 +1,93 @@
+// this will need to be updated later to check for database entry
+import { Installation } from "models/installation";
+import { GitHubServerApp } from "models/git-hub-server-app";
+import { GitHubInstallationClient } from "../github/client/github-installation-client";
+import { getCloudInstallationId } from "../github/client/installation-id";
+import { booleanFlag, BooleanFlags } from "config/feature-flags";
+import { GitHubUserClient } from "../github/client/github-user-client";
+import Logger from "bunyan"
+import { GitHubAppClient } from "../github/client/github-app-client";
+
+export const GITHUB_ENTERPRISE_CLOUD_HOSTNAME = "https://github.com";
+export const GITHUB_ENTERPRISE_CLOUD_API_BASEURL = "https://api.github.com";
+export const GITHUB_ENTERPRISE_CLOUD_ACCEPT_HEADER = "application/vnd.github.v3+json"
+
+export interface GitHubEnterpriseUrls {
```

Comment:
Not really happy with the name of this one seeing the acceptHeader isn't really a url... suggestions?